### PR TITLE
Fixes #252 :Added more social icons

### LINKED ISSUE #252: Social links to be added in some places
##Working On it

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #252

### CODE REVIEW

**Comment by saarthakchats** [comment]
https://saarthakchats.github.io/labs.fooooososossasia.org/

**Comment by saarthakchats** [comment]
@ParthS007 
I think it's fine now

File: index.html [hunk 1]
```diff
@@ -163,9 +163,13 @@
 									<li><a href="http://twitter.com/fossasia" target="_self"><i class="icon social_twitter"></i></a></li>
 									<li><a href="http://facebook.com/fossasia" target="_self"><i class="icon social_facebook"></i></a></li>
 									<li><a href="http://fossasia.net" target="_self"><i class="icon social_dribbble"></i></a></li>
-               					    <li><a href="https://www.youtube.com/channel/UCQprMsG-raCIMlBudm20iLQ" target="_self"><i class="icon social_youtube icon-large"></i></a></li>
-                 					<li><a href="https://www.flickr.com/photos/fossasia/" target="_self"><i class="icon social_flickr"></i></a></li>
-                					<li><a href="https://www.instagram.com/explore/tags/fossasia/" target="_self"><i class="icon social_instagram"></i></a></li>
+               					    			<li><a href="https://www.youtube.com/channel/UCQprMsG-raCIMlBudm20iLQ" target="_self"><i class="icon social_youtube icon-large"></i></a></li>
```
Comment (abishekvashok) [inline index.html]: This line was overintended

File: index.html [hunk 2]
```diff
@@ -163,9 +163,13 @@
...
+                 							<li><a href="https://www.flickr.com/photos/fossasia/" target="_self"><i class="icon social_flickr"></i></a></li>
+                							<li><a href="https://www.instagram.com/explore/tags/fossasia/" target="_self"><i class="icon social_instagram"></i></a></li>
+                							<li><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github fa-lg"></i></a></li>
+                							<li><a target="_self" href="https://www.linkedin.com/groups/FOSSASIA-3762811"><i class="icon social_linkedin icon-large"></i></a></li>
+                							<li><a target="_self" href="https://plus.google.com/108920596016838318216"><i class="icon fa fa-google-plus"></i></a></li>
+									<li><a target="_self" href="blog.fossasia.org"><i class="fa fa-external-link" aria-hidden="true"></i></a></li>
```
Comment (akshatnitd) [inline index.html]: Blog link doesn't work. And fix icons alignment as suggested by others!

**Pull request closed without merging**